packages/flutter/lib/src/material/button.dart

```diff
@@ -77,22 +111,33 @@ abstract class MaterialButton extends StatefulWidget {
   }
 }
 
+/// A state object for [MaterialButton].
+///
+/// Subclasses of [MaterialButton] should use a subclass of
+/// [MaterialButtonState] for their state objects.
 abstract class MaterialButtonState<T extends MaterialButton> extends State<T> {
+  /// Whether this button is in the process of potentially being pressed.
   bool highlight = false;
 
+  /// The height at which to place this button.
```

Comment:
Elevation is the obvious word, but we try to use different words in the dartdoc to give some hint about what these jargon terms mean.  I've switch these to `z-coordinate`.